app/SearchApi/SearchApi.Web/People/Identifier.cs

```diff
@@ -0,0 +1,36 @@
+﻿using Newtonsoft.Json;
+using System;
+using System.Collections.Generic;
+using System.ComponentModel;
+using System.Linq;
+using System.Threading.Tasks;
+
+namespace SearchApi.Web.People
+{
+    [Description("Represents a set of information to execute a search on a person")]
+    public class Identifier
+    {
+        [JsonConstructor]
+        public Identifier()
+        {
+        }
+
+        [Description("The identification serial number of this identifier.")]
+        public string Identification { get; set; }
+
+        [Description("The identification effective date.")]
+        public DateTime? IdentificationEffectiveDate { get; set; }
+
+        [Description("The identification expiration date.")]
+        public DateTime? IdentificationExpirationDate { get; set; }
+
+        [Description("The identfication type,such as driver license.")]
+        public int IdentifierType { get; set; }
```

Comment:
this should be an enum or a class, the int is insufficient to resolve the identifier

Reply:
agree